daemon/container_operations_windows.go

```diff
@@ -3,8 +3,16 @@
 package daemon
 
 import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/cloudflare/cfssl/log"
```

Comment:
There really is no reason for this - I is a genuine bug in the Unix implementation as far as I can tell. There's no reason to introduce a logging package which isn't used on Windows currently. It should be a logrus.Errorf otherwise it simply won't be caught. Can you update the unix file in this PR? I